### DISCUSSION TITLE
Name change e.g. Rust Patterns for Dart

### DISCUSSION THREAD

**Discussion by tk512** [mcmah309/rust#4] [Ideas]
I think the current name is confusing. A name like "rust_core" makes me think it's either a type of embedded Rust inside Dart, or even Rust libraries that have been ported. In reality it's rather about Rust patterns being available in Dart and Flutter. I think a name change would provide a lot of value here and lessen the confusion. I really think this library has a lot of potential. Well done on the work.

**Comment by mcmah309** [comment]
I can see that. Unfortunately I think it may be a bit late for a name change, the package is already on [pub.dev](https://pub.dev/packages/rust_core) as "rust_core". Maybe I can change some other things to make it more apparent?

maybe "rust_core is an implementation of Rust's Core Library in Dart, bringing the power of Rust to Dart!" to "rust_core is a pure Dart implementation of patterns found in the Rust Core Library, bringing the power of Rust to Dart!"

Other ideas welcome.

**Comment by tk512** [comment]
Yeah. Great idea!On 05 Jul 2024, at 16:44, Henry ***@***.***> wrote:﻿
I can see that. Unfortunately I think it may be a bit late for a name change, the package is already on pub.dev as "rust_core". Maybe I can change some other things to make it more apparent?
maybe "rust_core is an implementation of Rust's Core Library in Dart, bringing the power of Rust to Dart!" to "rust_core is an implementation of patterns found in the Rust Core Library, bringing the power of Rust to Dart!"
Other ideas welcome.

**Comment by mcmah309** [comment]
This project has been renamed to [rust](https://pub.dev/packages/rust) (dropping the `_core`)
